### ISSUE TITLE
Tao Feng [Tao Feng Fist Of The Lotus] [MS-020] [1.03]

### ISSUE THREAD

**Issue by kennethedmonds** [Cxbx-Reloaded/game-compatibility#602]
### Description

After I click start a black screen appears and immediately an error follows. I was able to press ignore to see another error. Two different errors are available.

```
---------------------------
Cxbx-Reloaded
---------------------------
Received Breakpoint Exception (int 3) @ EIP := 0x002B18B0(=unknown+0x0)

  Press Abort to terminate emulation.
  Press Retry to debug.
  Press Ignore to continue emulation.
---------------------------
Abort   Retry   Ignore   
---------------------------
---------------------------
Cxbx-Reloaded
---------------------------
Received Breakpoint Exception (int 3) @ EIP := 0x002B194B(=unknown+0x0)

  Press Abort to terminate emulation.
  Press Retry to debug.
  Press Ignore to continue emulation.
---------------------------
Abort   Retry   Ignore   
---------------------------
```

### Dumps 

[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1669606/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1669607/KrnlDebug.txt)
[Tao Feng-3b20c003.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1669608/Tao.Feng-3b20c003.txt)

### Status
**Previous behavior:** 

None reported

**Current behavior:**

Crashes with error.

### Additional Information

Tested on cc0e5105 (Jan 19 2018)

**Comment by HyperScorpio** [comment]
Error confirmed.

**Comment by SonicThePS4** [comment]
![image](https://user-images.githubusercontent.com/39503785/48678398-e5327380-eb50-11e8-85d0-6edf5b3d9753.png)

Spoiler alert! we're making some progress! :smiley:

**Comment by HyperScorpio** [comment]
Hot womens soon.

**Comment by SonicThePS4** [comment]
@HyperScorpio You know it Pal Jade Dragon is going to be my main :smile:

**Comment by SonicThePS4** [comment]
Quick little progress! It made it to the Press Start Screen Title and I heard the playing the Theme music while the dirty disc dirty/damage screen appeared! :grinning:  

![image](https://user-images.githubusercontent.com/39503785/56428816-c0bcfd80-628e-11e9-8137-40362cc20be5.png)

**Comment by HyperScorpio** [comment]
Still Black Screen but...
If you press A slowly one by one you can bypass the menu and listen to the VS and the charather selection, stage select and after that, the green error appears. At least we have some tiny improvements.

**Comment by SonicThePS4** [comment]
@HyperScorpio Yep, that's what I heard too. When I heard it on my PC, I got so excited just from that lol. Is going to be insane.

**Comment by 5t6y7uh8i9o** [comment]
https://youtu.be/q4bDeF5duHU
